include/libjsonnet.h

```diff
@@ -31,7 +31,7 @@ limitations under the License.
  *
  * If this isn't the sae as jsonnet_version() then you've got a mismatched binary / header.
  */
-#define LIB_JSONNET_VERSION "v0.20.0"
+#define LIB_JSONNET_VERSION "0.20.0"
```

Comment:
This is the version of the jsonnet interpreter. I don't think you can change this.